src/kirby/components/page/page.component.ts

```diff
@@ -161,30 +165,41 @@ export class PageComponent implements OnInit, OnDestroy, AfterContentInit, After
   ) {}
 
   ngOnInit(): void {
+    this.url = this.router.url;
+
     this.removeWrapper();
     // TODO JEO / AFL Fix page title observer...
-    // this.routerEventsSubscription = this.router.events.subscribe((event: RouterEvent) => {
-    //   if (event instanceof NavigationStart) {
-    //     this.toolbarTitleVisibility = 'hidden';
-    //     this.toolbarStickyActionsVisibility = 'hidden';
-    //   }
-    //   if (event instanceof NavigationEnd && this.pageTitleObserver) {
-    //     this.pageTitleObserver.disconnect();
-    //     this.pageTitleObserver = this.observePageTitle();
-    //   }
-    // });
+    this.routerEventsSubscription = this.router.events.subscribe((event: RouterEvent) => {
+      if (event instanceof NavigationStart && event.url !== this.url) {
+        this.pageTitleObserver.unobserve(this.pageTitle.nativeElement);
+      }
+      if (event instanceof NavigationEnd && event.url === this.url) {
+        this.pageTitleObserver.observe(this.pageTitle.nativeElement);
+      }
+    });
+  }
+
+  ngOnChanges(changes: SimpleChanges): void {
+    if (changes.title) {
+      this.initializeTitle();
```

Comment:
Is this enough / needed? Or do we need to execute the stuff inside `ngAfterViewInit` again?